src/storage/mvcc/txn.rs

```diff
@@ -266,6 +275,13 @@ impl<S: Snapshot> MvccTxn<S> {
                 }
             }
             _ => {
+                // `get_txn_commit_info` will scan write cf in reversed order, so we use
```

Comment:
I think this block should be moved to L310.

Reply:
It's better to detect collision before calling `get_txn_commit_info` to minimize the overhead, because `get_txn_commit_info` can move the write cursor to a position before `key_startts`. If we do the collision then, `reverse_seek` of rocksdb will be invoked twice.

Reply:
Ok.. I've changed